node/stakeValuesProcessor/disabledStakeValuesProcessor.go

```diff
@@ -0,0 +1,20 @@
+package stakeValuesProcessor
```

Comment:
you can get rid of the code cov issues if you place this file in ./disabled package

Reply:
done.